src/AppInstallerRepositoryCore/Public/AppInstallerRepositorySource.h

```diff
@@ -101,6 +101,9 @@ namespace AppInstaller::Repository
     // Gets the details for a single source.
     std::optional<SourceDetails> GetSource(std::string_view name);
 
+    // Gets the details for a single source by identifier
+    std::optional<SourceDetails> GetSourceByIdentifier(std::string_view identifier);

```

Comment:
>std::string_view identifier [](http://example.com/codeflow?start=55&length=27)

The way that this is being used, it should be a `PredefinedSource`.  If that is confusing because not all values are being exposed via COM, then another enum name would be fine (although I don't think that is necessary).

Reply:
This got removed.

Reply:
Appears to still be here in the latest commit.